src/components/ui/DatePicker/DatePicker.svelte

```diff
@@ -0,0 +1,345 @@
+<script lang="ts">
+  import Calendar from '@nasa-jpl/stellar/icons/svg/calendar.svg?component';
+  import ChevronLeft from '@nasa-jpl/stellar/icons/svg/chevron_left.svg?component';
+  import ChevronRight from '@nasa-jpl/stellar/icons/svg/chevron_right.svg?component';
+  import { createEventDispatcher, onDestroy, onMount } from 'svelte';
+  import { createPopperActions } from 'svelte-popperjs';
+
+  import { getTarget } from '../../../utilities/generic';
+  import { getDoyTime, parseDateTime } from '../../../utilities/time';
+  import DatePickerDropdown from './DatePickerDropdown.svelte';
+  import Month from './Month.svelte';
+
+  const currentDate = new Date();
+  currentDate.setUTCHours(0);
+  currentDate.setUTCMinutes(0);
+  currentDate.setUTCSeconds(0);
+  currentDate.setUTCMilliseconds(0);
+  const currentYear = currentDate.getUTCFullYear();
+
+  export let dateString: string = '';
```

Comment:
Can we move the `export` under the `import`. General ordering: imports, exports, const, lets, reactive expressions, lifecycle functions, functions. At some point we need a style doc for these (currently) un-lintable things.

Reply:
You're ok with the exports being broken up a bit? The `currentDate` declaration can't be moved below because `min/maxDate` uses it.